js/pihole/index.js

```diff
@@ -182,36 +182,30 @@ function objectToArray(p){
 // Functions to update data in page
 
 function updateSummaryData(runOnce) {
-    $.getJSON("api.php?summary", function LoadSummaryData(data) {
-        //$("h3.statistic").addClass("glow");
-        if ($("h3#ads_blocked_today").text() != data.ads_blocked_today) {
-            $("h3#ads_blocked_today").addClass("glow");
-        }
-        if ($("h3#dns_queries_today").text() != data.dns_queries_today) {
-            $("h3#dns_queries_today").addClass("glow");
-        }
-        if ($("h3#ads_percentage_today").text() != data.ads_percentage_today) {
-            $("h3#ads_percentage_today").addClass("glow");
+    var setTimer = function(timeInMs) {
+        if (!runOnce) {
+            setTimeout(updateSummaryData, timeInMs);
         }
+    };
+    $.getJSON("api.php?summary", function LoadSummaryData(data) {
 
-        window.setTimeout(function(){
+        ["ads_blocked_today", "dns_queries_today", "ads_percentage_today"].forEach(function(today) {
+            var todayElement = $("h3#" + today);
+            todayElement.text() !== data[today] && todayElement.addClass("glow");
+        });
+
+        window.setTimeout(function() {
             $("h3#ads_blocked_today").text(data.ads_blocked_today);
             $("h3#dns_queries_today").text(data.dns_queries_today);
             $("h3#domains_being_blocked").text(data.domains_being_blocked);
             $("h3#ads_percentage_today").text(data.ads_percentage_today + "%");
             $("h3.statistic.glow").removeClass("glow")
         }, 500);
-
-        updateSessionTimer();
```

Comment:
We use it to reset the timer in the top right corner menu showing how long the session will be valid. It needs to be updated every time we make a request (as this will update the session).